src/shark/utilityconsole/util/searching/ParameterCriterion.java

```diff
@@ -31,13 +31,19 @@ public enum Criteria {
              * Actually maps to both {@link com.fs.starfarer.api.combat.WeaponAPI.WeaponType} and {@link com.fs.starfarer.api.combat.WeaponAPI.WeaponSize}
              */
             WEAPON_WITH_SIZE,
-            SHIP_PARAMETER
+            SHIP_PARAMETER,
+            WEAPON_PARAMETER
```

Comment:
do we want to keep this in `ParameterCriterion` or are two versions of `ParameterCriterion` a better option? One for just `findships` queries, and one for just `findweapons` queries rather than using the shared one?

Somehow i think the shared one is better. But am not sure.

Reply:
it's good. i'm not changing it now. resolving.